!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  coeff_distance_to_goal: 0.6
  coeff_distance_to_obstacle: 2.0
  collision_free: 0.0
  enable_drive_backward: true
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      coeff_distance_to_goal: 0.6
      coeff_distance_to_obstacle: 2.0
      collision_free: 0.0
      enable_drive_backward: true
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      guarantee_heading: false
      heuristic_cost: 1.0
      id: 0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      reverse_cost: 5.0
      show_flag: true
      state: true
      steer_change_cost: 2.5
      type: ''
    state: []
  guarantee_heading: false
  heuristic_cost: 1.0
  reverse_cost: 5.0
  show_flag: true
  steer_change_cost: 2.5
state: []
